eth/precompiles/modexp.py

```diff
@@ -64,14 +68,15 @@ def _extract_lengths(data: bytes) -> Tuple[int, int, int]:
     return base_length, exponent_length, modulus_length
 
 
-def _compute_modexp_gas_fee(data: bytes) -> int:
-    base_length, exponent_length, modulus_length = _extract_lengths(data)
+@curry
+def _compute_modexp_gas_fee_eip_198(data: bytes) -> int:
```

Comment:
Don't think this one needs it.
```suggestion
def _compute_modexp_gas_fee_eip_198(data: bytes) -> int:
```